includes/class-order.php

```diff
@@ -248,29 +541,38 @@ public static function update_status( $order_id, $response ) {
 		Zotapay::getLogger()->info( $message );
 
 		// Awaiting statuses.
-		if ( in_array( $response->getStatus(), array( 'CREATED', 'PENDING', 'PROCESSING' ), true ) ) {
+		if ( in_array( $response['status'], array( 'CREATED', 'PENDING', 'PROCESSING' ), true ) ) {
 			$note = sprintf(
 				// translators: ZotaPay status.
 				esc_html__( 'ZotaPay status: %s.', 'zota-woocommerce' ),
-				sanitize_text_field( $response->getStatus() )
+				sanitize_text_field( $response['status'] )
```

Comment:
Suggestion:
`esc_html()` should be used here for escaping the value, I suppose `sanitize_text_field()` is used earlier when creating the `$response` array.

When I think about it, maybe we shouldn't escape the note before adding to the database, it should be escaped when output on the screen by WooCommerce.

Reply:
Sure..